src/language-server/settings.ts

```diff
@@ -47,6 +63,62 @@ export default class Settings extends AutoDisposable {
       this.latest = {...Settings.defaults, ...settings};
       fire({newer: this.latest, older: previous});
     });
+    this.fileSynchronizer = fileSynchronizer;
+
+    this._disposables.push(
+        this.fileSynchronizer.fileChanges.listen((changes) => {
+          for (const change of changes) {
+            const workspacePath = converter.getWorkspacePathToFile(change);
+            if (workspacePath === 'polymer.json') {
```

Comment:
could there be cases where `workspacePath` is something like `../../polymer.json`? 
Would it be better to check for `workspacePath.endsWith('polymer.json')`?

Reply:
Currently all of our tools expect `polymer.json` to always be at the root directory of the project, and for the moment we expect that the user will open their editor's workspace to the same root directory.

This will get more complicated later, as we adapt to the internal google layout, but this is a good start.

When dealing with the FSUrlLoader as we are here `polymer.json` is based out of the workspace root.

Reply:
Understood, thanks for the info! 👌